src/foam/parse/SinkParser.java

```diff
@@ -26,12 +26,12 @@
   protected ClassInfo info_;
 
   public SinkParser(ClassInfo info) {
+    info_ = info;
```

Comment:
We should re-write this to not extend Alt but to rather just compose an Alt object.

```
public class SinkParser extends ProxyParser
{
  public SinkParser() {
    super(new Alt(starParser(), expressionListParser()));
  }
}
```